deb/Makefile

```diff
@@ -47,11 +48,6 @@ clean: ## remove build artifacts
 	$(RM) -r debbuild
 	[ ! -d sources ] || $(CHOWN) -R $(shell id -u):$(shell id -g) sources
 	$(RM) -r sources
-	$(RM) engine-image
```

Comment:
We need to discuss if we can already backport these changes

Reply:
Sure, that the main objective of this PR, what should we pick up to be close as possible of the nightly build without breaking the existing release process for previous version.